cmd/lotus-shed/verifreg.go

```diff
@@ -29,11 +22,6 @@ var verifRegCmd = &cli.Command{
 	Flags: []cli.Flag{},
 	Subcommands: []*cli.Command{
 		verifRegAddVerifierCmd,
-		verifRegVerifyClientCmd,
-		verifRegListVerifiersCmd,
-		verifRegListClientsCmd,
-		verifRegCheckClientCmd,
-		verifRegCheckVerifierCmd,
```

Comment:
This will break existing scripts, maybe we should keep/alias those commands here as hidden/with a deprecated warning?

Reply:
Ack